diff --git a/lib/esm/CalledWithFn.d.ts b/lib/esm/CalledWithFn.d.ts
index 8bfba11ba18cb132ee5da8ad208391fa035c5eae..a307f2aabd6b808aaa35aca9b20401df1ee637f1 100644
--- a/lib/esm/CalledWithFn.d.ts
+++ b/lib/esm/CalledWithFn.d.ts
@@ -1,6 +1,6 @@
 import { type Mock } from 'vitest';
-import type { CalledWithMock } from './Mock';
-import type { CalledWithImplementation, FallbackImplementation } from './types';
+import type { CalledWithMock } from './Mock.js';
+import type { CalledWithImplementation, FallbackImplementation } from './types.js';
 type CalledWithFnArgs<Y extends any[], T> = {
     fallbackMockImplementation?: FallbackImplementation<Y, T>;
     mock?: Mock<CalledWithImplementation<Y, T>>;
diff --git a/lib/esm/Mock.d.ts b/lib/esm/Mock.d.ts
index 0e9122c7d88a8802976e53fa2fcd5c299bf8dd13..e3324201ec8af13a103f3ea49fd5b72db7414324 100644
--- a/lib/esm/Mock.d.ts
+++ b/lib/esm/Mock.d.ts
@@ -1,7 +1,7 @@
 import type { DeepPartial } from 'ts-essentials';
 import { type Mock } from 'vitest';
-import type { MatchersOrLiterals } from './Matchers';
-import type { FallbackImplementation } from './types';
+import type { MatchersOrLiterals } from './Matchers.js';
+import type { FallbackImplementation } from './types.js';
 type ProxiedProperty = string | number | symbol;
 interface GlobalConfig {
     ignoreProps?: ProxiedProperty[];
diff --git a/lib/esm/index.d.ts b/lib/esm/index.d.ts
index e90bfe9923944f406b229599eac6d66bb7ea2d50..4bd6d61cac0143a87209171082dfa41eb1987f74 100644
--- a/lib/esm/index.d.ts
+++ b/lib/esm/index.d.ts
@@ -1,5 +1,5 @@
-export * from './CalledWithFn';
-export * from './Matchers';
-export type * from './Mock';
-export * from './Mock';
+export * from './CalledWithFn.js';
+export * from './Matchers.js';
+export type * from './Mock.js';
+export * from './Mock.js';
 //# sourceMappingURL=index.d.ts.map
